config/routes.rb

```diff
@@ -257,6 +257,10 @@
           get 'student_interface'
           post 'execute_test_run'
         end
+
+        member do
```

Comment:
It seems like you are reusing an existing route, so you shouldn't add a new route here.